cmd/logger/main.go

```diff
@@ -0,0 +1,99 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"github.com/kubeflow/kfserving/pkg/apis/serving/v1alpha2"
+	"net/http"
+	"net/url"
+	"os"
+	"sigs.k8s.io/controller-runtime/pkg/runtime/signals"
+
+	"github.com/kubeflow/kfserving/pkg/logger"
+	"github.com/pkg/errors"
+	"golang.org/x/net/http2"
+	"golang.org/x/net/http2/h2c"
+	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
+)
+
+var (
+	logUrl    = flag.String("log-url", "", "The URL to send request/response logs to")
+	port      = flag.String("port", "8080", "Logger port")
+	svcHost   = flag.String("svc-host", "0.0.0.0", "Service host")
```

Comment:
I hate to nit here, but we've been calling "svc" as "component" in other areas of the code.